homeassistant/components/sensor/cert_expiry.py

```diff
@@ -0,0 +1,98 @@
+"""Counts the days an HTTPS (TLS) certificate will expire (days).
+
+Example configuration.yaml:
+
+  sensor:
+    - platform: cert_expiry
+      server_name: home-assistant.io
+
+For more details about this sensor please refer to the
+documentation at https://home-assistant.io/components/sensor.cert_expiry
+"""
+import logging
+import ssl
+import socket
+import datetime
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.util import Throttle
+from homeassistant.helpers.entity import Entity
+
+REQUIREMENTS = []
+_LOGGER = logging.getLogger(__name__)
+
+CONF_SERVER_NAME = 'server_name'
+CONF_SERVER_PORT = 'server_port'
```

Comment:
We already have `host` and `port` for those two configuration variables which are defined in `const.py`.